addons/sale/models/sale_order_line.py

```diff
@@ -1021,6 +1060,19 @@ def _compute_untaxed_amount_to_invoice(self):
 
             line.untaxed_amount_to_invoice = amount_to_invoice
 
+    @api.depends('discount', 'price_total', 'product_uom_qty', 'qty_delivered', 'qty_invoiced_posted')
+    def _compute_amount_to_invoice(self):
+        to_compute = self.company_id.account_use_credit_limit
```

Comment:
Since it's not in the view, I wouldn't filter out `account_use_credit_limit`.

In practice, this field will only be computed when used in other functions, and thos functions already filter based on the credit limit

Reply:
It was just an extra precaution, to be safe, but if you think it's not necessary, I can remove it :wink: